amundsen_application/api/utils/notification_utils.py

```diff
@@ -53,9 +53,10 @@ def get_notification_html(*, notification_type: str, options: Dict, sender: str)
     Returns the formatted html for the notification based on the notification_type
     :return: A string representing the html markup to send in the notification
     """
-    resource_url = options.get('resource_url')
-    if resource_url is None:
-        raise Exception('resource_url was not provided in the notification options')
+    resource_path = options.get('resource_path')
+    if resource_path is None:
+        raise Exception('resource_path was not provided in the notification options')
+    resource_url = '{url_base}{resource_path}'.format(resource_path=resource_path, url_base=app.config['FRONTEND_BASE'])
```

Comment:
Yes since we control the creation of both.`url_base` is defined in the configs and the follows the current pattern of *not* having a trailing slash, and the code that generates `resource_url` as a leading slash. 

However since this is susceptible to human error, how about we raise an exception if `url_base` has a trailing `/` or if `resource_path` does not begin with a `/`?